velox/common/memory/MallocAllocator.cpp

```diff
@@ -102,6 +104,7 @@ bool MallocAllocator::allocateNonContiguous(
 
   // Successfully allocated all pages.
   numAllocated_.fetch_add(mix.totalPages);
+  numMapped_.fetch_add(mix.totalPages);
```

Comment:
This is not mapped pages in MallocAllocator? This should only be set for MmapAllocator? Thanks!